BrightID/src/components/NewConnectionsScreens/SuccessScreen.js

```diff
@@ -23,6 +25,27 @@ import { sortByDateAddedDescending } from '../Connections/models/sortingUtility'
 
 export const SuccessScreen = ({ navigation }) => {
   const dispatch = useDispatch();
+  // clear navigation history to prevent going back to confirmation and preview screens with back button
+  const resetNav = () => {
+    navigation.reset({
+      index: 1,
+      routes: [
+        { name: 'Home' },
+        {
+          name: 'Connections',
+        },
+      ],
+    });
+    return true;
+  };
+
+  useFocusEffect(
+    useCallback(() => {
+      BackHandler.addEventListener('hardwareBackPress', resetNav);
+      return () =>
+        BackHandler.removeEventListener('hardwareBackPress', resetNav);
+    }, []),
```

Comment:
Similar problem with missing dependency in useCallback like the `reject` function in PreviewConnectionScreen. In this case `resetNav` dep is missing.

Reply:
again, resetNav should *probably* be placed outside of the component, but it's only dependency is navigation, which should not change between renders